test/integration/resolver/package.json

```diff
@@ -1,9 +1,11 @@
-{
-  "name": "resolver",
-  "private": true,
-  "alias": {
-    "aliased": "foo",
-    "aliased-file": "./bar.js",
-    "something": "./nested/test.js"
-  }
-}
+{

```

Comment:
Can this be cleaned up? Seems like you have changed the formatting of the file or something, so it now overwrites the entire file

Reply:
should be fixed now: wrong line endings :/